packages/examples/fortune-v2/job-launcher/server/src/common/decorators/user.ts

```diff
@@ -0,0 +1,15 @@
+import { createParamDecorator, ExecutionContext } from "@nestjs/common";
+import { UserStatus, UserType } from "../enums/user";
+import { IBase } from "./base";
+
+export interface IUser extends IBase {
+  password: string;
+  email: string;
+  status: UserStatus;
+  type: UserType;
+}
+
```

Comment:
/**
 * Custom decorator to retrieve the currently logged-in user.
 * Returns the user object from the request context.
 */